definition-map.js

```diff
@@ -75,7 +75,56 @@ const QUERIES = {
             includeCorrectAnswers: "Flag to include questions the user previously got correct.",
             quizzes: "The IDs of the Quizzes user selects."
         }
-    }
+    },
+    Forums: {
+        metadescription: "This query returns a forum object. A forum is any area you can add a discussion to. For example, assignments, forum page type, discussions below the course on each section.",
+        args: {
+            courseId: "The ID of the course. This arguement is required", 
+            clientId: "The ID of the client"
+        }
+    },
+    Threads: {
+        metadescription: "This query returns a list of threads.",
+        args: {
+            courseId: "The ID of the course", 
+            clientId: "The ID of the client",
+            forumId: "The ID of the forum. This argument is required",
+            commentableType: "A commentable type can be thread, widgetThread, assignmentSubmission, assignment or a discussionBoard",
+            page: "The number of the page to be returned",
```

Comment:
`augment-schema.js` should automatically fill in the description for any args named `page`: 

https://github.com/thoughtindustries/helium-graphql/blob/3a10536a4b1cdc39ac7755aaf7f27b2dff49d92c/augment-schema.js#L86

Can probably add something there to handle the `perPage` field as well since that's fairly common.

Reply:
`else if` added to `augment-schema.js` to handle `perPage`.

Does this mean that I remove `page` and `perPage` from `definition-map`?

Reply:
@AlexONeillTI Yup, should be good to remove them from the `definition-map` after. You should be able to run `npm run dev` and verify the changes at `localhost:4400`